src/config/dynamic-theme-fixes.config

```diff
@@ -19441,6 +19441,19 @@ INVERT
 
 ================================
 
+s.weibo.com
+
+CSS
+[data-darkreader-inline-bgcolor] {
```

Comment:
Hi, could you please try targeting those exact nodes by classes or other attributes? For example something like:
```
.class {
    background-color: var(--darkreader-neutral-background) !important;
    border-color: ${#cccccc} !important;
}
```

Reply:
done🫡